easybuild/easyconfigs/x/xtb/xtb-6.3.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
```

Comment:
@dplacencia Looks like the official software name is `xTB`, so we should stick to that?

Also, do let us know if you need help with making the test suite pass.

Reply:
I am having problems with the test suite, a problem in the dependency graph is jumping. However, at the end it says "No conflicts detected"